### ISSUE TITLE
Error Generating Video

### ISSUE THREAD

**Issue by ZOTUPsrl** [patrickmatte/scroll-capture#39]
Some website doesn't let me complete and download the video. Most of the website are done on Laravel

**Comment by ZOTUPsrl** [comment]
I see this:
sample_video.webm: Invalid data found when processing input

FFMPEG_END

**Comment by patrickmatte** [comment]
Do you have any urls that you can share so I can try to reproduce this?
Also, if you can copy and paste here the complete output, that could also be helpful.
Thanks!

**Comment by ZOTUPsrl** [comment]
https://www.gfstudio.com
![Screenshot 2024-03-18 alle 17 14 38](https://github.com/patrickmatte/scroll-capture/assets/160721705/dfae3d18-0a34-4dd5-9cc6-a444fd7df627)
![Screenshot 2024-03-18 alle 17 14 55](https://github.com/patrickmatte/scroll-capture/assets/160721705/b9f1d6df-edf7-4ec5-a7de-f5cad7424640)
![Screenshot 2024-03-18 alle 17 14 49](https://github.com/patrickmatte/scroll-capture/assets/160721705/45dfca87-5947-43d5-b9f2-6298027138e0)

**Comment by patrickmatte** [comment]
I tried your URL, the error shows up in the javascript console

`Refused to load media from 'blob:https://www.gfstudio.com/5dca48ab-1062-4f08-9d8d-96120d2c4cf8' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'media-src' was not explicitly set, so 'default-src' is used as a fallback.`

First time I see this error. I'm not sure how to fix it. I also see this other error in the console when I loaded your URL
`Error with Feature-Policy header: Some features are specified in both Feature-Policy and Permissions-Policy header: accelerometer, camera, geolocation, gyroscope, magnetometer, microphone, payment, usb. Values defined in Permissions-Policy header will be used.`

Maybe it's those policy header on your server that are blocking the blob from loading. I don't know, I'm not very familiar with that kind of stuff, servers, content-security policies etc...

**Comment by prowhiz** [comment]
I have the same issue on this page: https://freedom.lightspuncreative.com
In the console, this is the error I get:

![image](https://github.com/patrickmatte/scroll-capture/assets/25750427/d77251a7-9bad-4aa1-9b5d-c638883c4761)

**Comment by prowhiz** [comment]
> I have the same issue on this page: https://freedom.lightspuncreative.com In the console, this is the error I get:
> 
> ![image](https://private-user-images.githubusercontent.com/25750427/318016363-d77251a7-9bad-4aa1-9b5d-c638883c4761.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3MDYxNTAsIm5iZiI6MTcxMTcwNTg1MCwicGF0aCI6Ii8yNTc1MDQyNy8zMTgwMTYzNjMtZDc3MjUxYTctOWJhZC00YWExLTliNWQtYzYzODg4M2M0NzYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDA5NTA1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkyZGNkNGI2ODUyNTdlZjRmZDg2NzYxZWNmMmFhMGU0NmE5NzRkOWQ4MjY5YTU1ZDMwOTI5ODUwODA1YzE0OTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.md3czIHACG9-0rK9AA1NB0yAvM6bE-rpYqso_ATlzmY)

[UPDATE]
I had been trying to capture on an incognito window (I'm using microsoft edge chromium Version 122.0.2365.92 (Official build) (64-bit))

Now I tried it on a regular window and the website was captured and video saved successfully

**Comment by patrickmatte** [comment]
@prowhiz Thanks for reporting this, I had never tried to use the extension inside an incognito window. At first I didn't even see the extension button, I had to go to the extension settings to manually enable the extension for incognito mode. And you are right, I'm afraid the video capture fails for ay website viewed inside an incognito window. I will need to investigate. It is a different issue from the one above so I created another issue issue. https://github.com/patrickmatte/scroll-capture/issues/42
